proxy/src/transport/tls/cert_resolver.rs

```diff
@@ -32,14 +32,19 @@ impl CertResolver {
     /// TODO: Verify that the public key of the certificate matches the private
     /// key.
     pub fn new(
-        trust_anchors: &webpki::TLSServerTrustAnchors,
+        root_cert_store: &rustls::RootCertStore,
         cert_chain: Vec<rustls::Certificate>,
         private_key: untrusted::Input)
         -> Result<Self, config::Error>
     {
         let now = webpki::Time::try_from(SystemTime::now())
             .map_err(|ring::error::Unspecified| config::Error::TimeConversionFailed)?;
 
+        let trust_anchors = root_cert_store.roots.iter()
+            .map(|owned_trust_anchor| owned_trust_anchor.to_trust_anchor())
+            .collect::<Vec<_>>();
```

Comment:
It would be nice if we could just pass `&root_cert_store.roots[..]` to `webpki::TLSServerTrustAnchors`, but based on a quick perusal of the `webpki` and `rustls` documentation, it seems like that wouldn't be possible without changes in those crates?